region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -1194,15 +1194,19 @@
         "HiJump",
         "canHorizontalDamageBoost",
         "canTrickyJump",
-        "canCrouchJump",
+        {"or": [
+          "canCrouchJump",
+          "canInsaneJump"
```

Comment:
I think `h_trickyToCarryFlashSuit` may be justified here? Without a crouch jump, it's pretty precise.